fe1-web/src/features/evoting/components/__tests__/EventElection.test.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import { EventTags, Hash, Timestamp } from 'core/objects';
+import { mockLaoId, mockLaoIdHash, mockLaoName } from '__tests__/utils';
+import STRINGS from 'resources/strings';
+import {
+  Election,
+  ElectionStatus,
+  Question,
+  QuestionResult,
+  RegisteredVote,
+} from 'features/evoting/objects';
+import EventElection from '../EventElection';
+
+// #region test data initialization
```

Comment:
Huh, interesting. In VSCode it works like this and according to [this](https://www.jetbrains.com/help/rider/Coding_Assistance__Surrounding_with_Region.html#managing-regions-in-the-editor) it should also work in webstorm? Does it work for you without the `#`? Because then it at least does not work in VSCode (which would be fine for me, I don't mind scrolling a bit haha)

Reply:
Found this: https://github.com/madskristensen/WebEssentials2015/issues/103

Reply:
Oh no! Apparently `#region` works in webstorm but `# region` does not. And `eslint` wants spaces before a comment 😂

https://youtrack.jetbrains.com/issue/WEB-13948:

> //region
> // region
> //#region
> all work in 2019.2, but
> // #region
> does not.
> 
> This sucks because only the ones with # work with VS Code, and most eslint rulesets require a space after //, so there's no way to make everyone happy.

So I guess we can go with the very funny

```
region // #region NameOfRegion

// endregion // #endregion
```

Reply:
Hahaha okay I definitely didn't think it would be that much of a mess.
Does this work with VSCode using this method at least ?

Reply:
The `// #region` worked in VSCode but since Webstorm is the recommended IDE we should probably go with `// region x` and I will look for a plugin or something :)